packages/@aws-cdk/toolkit/lib/api/aws-cdk.ts

```diff
@@ -28,6 +28,11 @@ export { WorkGraph } from '../../../../aws-cdk/lib/util/work-graph';
 export type { Concurrency } from '../../../../aws-cdk/lib/util/work-graph';
 export { WorkGraphBuilder } from '../../../../aws-cdk/lib/util/work-graph-builder';
 export type { AssetBuildNode, AssetPublishNode, StackNode } from '../../../../aws-cdk/lib/util/work-graph-types';
+export { CloudWatchLogEventMonitor } from '../../../../aws-cdk/lib/api/logs/logs-monitor';
+export { findCloudWatchLogGroups } from '../../../../aws-cdk/lib/api/logs/find-cloudwatch-logs';
+
+// Test APIs
+export { MockSdk } from '../../../../aws-cdk/test/util/mock-sdk';
```

Comment:
Okay that import needs to go elsewhere. Apologies, I didn't explain the purpose of this file well enough.

Basically we are using it as bundling entrypoint. Everything in here will be bundled for distribution. We don't need to `MockSdk` bundled.

Reply:
your linter rule pointed me here :). but i'll make an exception for testing imports in that rule